### DISCUSSION TITLE
Readonly mode notification

### DISCUSSION THREAD

**Discussion by omicronn88** [provectus/kafka-ui#1766] [Ideas]
Hi guys. I really like what you're doing, but I was wondering maybe you could add a readonly notification to users when they try to perform write operations when cluster is added in a readonly mode? So that they know that it's on purpose.

**Comment by Haarolean** [comment]
Hey, thanks for reaching out. I'll convert this to an issue.

**Reply by omicronn88** [reply]
Thank you :)
